build/source/driver/multi_driver.f90

```diff
@@ -247,77 +357,123 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
  case default; call handle_err(30,'unable to identify vegetation category')
 end select
 
+! allocate space for default model parameters
+if(allocated(dparStruct%hru))then
```

Comment:
This check will never be activated with the current code organization. However, it is possible that someone may wist to allocate this multiple times at a later date (don't know why someone would do this). So the check is for safety reasons.